config/default.yml

```diff
@@ -31,6 +31,8 @@ Rails/ActiveRecordAliases:
                   Use `update!` instead of `update_attributes!`.
   Enabled: true
   VersionAdded: '0.53'
+  SafeAutoCorrect: false
+  AutoCorrect: false
```

Comment:
Ah, my explanation was bad 💦`AutoCorrect: false` is not necessary. It should be auto-corrected when `rubocop --safe-auto-correct` option is not specified.

Reply:
Fixed!